### ISSUE TITLE
[error] U+1EDB from UTF-8 to ISO-8859-1

### ISSUE THREAD

**Issue by wreckitkenny** [Ekultek/whitewidow#49]
[22:23:17 ERROR] Failed with error code U+1EDB from UTF-8 to ISO-8859-1
[22:23:17 FATAL] Program failed with error code: U+1EDB from UTF-8 to ISO-8859-1, error saved to error_log.txt
[22:23:17 FATAL] Issue template is being generated for this error, create a new issue named: IoPIg56 U+1EDB from UTF-8 to ISO-8859-1

##error_log
["/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/util.rb:77:in `encode'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/util.rb:77:in `from_native_charset'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:243:in `from_native_charset'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:237:in `block in proc_query'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:236:in `map'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:236:in `proc_query'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:299:in `block in build_query'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:298:in `each'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize/form.rb:298:in `build_query'", "/var/lib/gems/2.3.0/gems/mechanize-2.7.5/lib/mechanize.rb:587:in `submit'", "/home/kenny/Desktop/Scanning/SQL/whitewidow/lib/whitewidow/scanner.rb:41:in `get_urls'", "whitewidow.rb:64:in `<main>'"]

##issue

--

## Issue/Enhancement/Question (be specific)
Getting error: U+1EDB from UTF-8 to ISO-8859-1

## Exact error message/Enhancement information
U+1EDB from UTF-8 to ISO-8859-1

## Steps to reproduce if applicable or steps on what should be done
Run with {:default=>true}

## Search query if applicable (please use exact search query)
inurl:/index2.php?page=

## User agent (if applicable)
N/A

## Whitewidow version number (must have the actual version run `ruby whitewidow.rb --version`)
1.9.11.24

## Ruby version number (run ruby --version)
2.3.3

**Comment by Ekultek** [comment]
Well shit..

**Comment by Ekultek** [comment]
Can you give the exact URL where this error was given?

**Comment by Ekultek** [comment]
I can not reproduce this error, please tell me exact steps on how the error can be reproduced. The only thing I've been able to get is this:

```
[11:35:22 INFO] Parsing page for SQL syntax error: http://www.littlesaintsorphan
ages.org/index2.php?page=online AND 1=1
[11:35:23 ERROR] Failed with error code 406 Not Acceptable
[11:35:23 FATAL] Program failed with error code: 406 Not Acceptable, error saved
 to error_log.txt
[11:35:23 FATAL] Issue template is being generated for this error, create a new
issue named: aujCmko 406 Not Acceptable
```

Which is not your error. Either provide the exact URL, exact steps on how this was achieved, or specific steps given when the error was obtained. Otherwise this issue will be closed as non-reproducible.

**Comment by wreckitkenny** [comment]
I use -d option for searching a vulnerable website on Internet. And get the error which i mentioned at the beginning of this thread..

**Comment by Ekultek** [comment]
As I specified, I am unable to reproduce it, so what other options where you running along with the default flag?

**Comment by inzamamraja** [comment]
An error occurred while installing nokogiri (1.6.8), and Bundler cannot
continue.
Make sure that `gem install nokogiri -v '1.6.8'` succeeds before bundling.

**Comment by Ekultek** [comment]
@inzamamraja make a new issue

**Comment by Ekultek** [comment]
Found it

```
C:\Users\z-perkins-thomas\Documents\bin\ruby\whitewidow>ruby whitewidow.rb -d
 __      __.__    .__  __         __      __.__    .___
/  \    /  \  |__ |__|/  |_  ____/  \    /  \__| __| _/______  _  __
\   \/\/   /  |  \|  \   __\/ __ \   \/\/   /  |/ __ |/  _ \ \/ \/ /
 \        /|   Y  \  ||  | \  ___/\        /|  / /_/ (  <_> )     /
  \__/\  / |___|  /__||__|  \___  >\__/\  / |______ | \____/ \/\_/
       \/       \/              \/      \/           \/    v1.9.11.24(dev)

Clone: https://github.com/WhitewidowScanner/whitewidow.git
Downloads: https://github.com/WhitewidowScanner/whitewidow/releases
[ ! ] Use of this program for malicious intent is illegal. For more information
run the --legal flag

[08:09:47 INFO] I'm searching for possible SQL vulnerable sites, using search qu
ery ∩╗┐inurl:/index2.php?page=
[08:09:47 INFO] Using default user agent
[08:09:48 ERROR] Failed with error code U+2229 to ISO-8859-1 in conversion from
IBM437 to UTF-8 to ISO-8859-1
[08:09:48 FATAL] Program failed with error code: U+2229 to ISO-8859-1 in convers
ion from IBM437 to UTF-8 to ISO-8859-1, error saved to error_log.txt
[08:09:48 FATAL] Issue template is being generated for this error, create a new
issue named: zubOCXI U+2229 to ISO-8859-1 in conversion from IBM437 to UTF-8 to
ISO-8859-1
[08:09:48 INFO] An issue template has been generated for you and is located in C
:/Users/z-perkins-thomas/Documents/bin/ruby/whitewidow/tmp/issues/Xgb2Noi.txt

C:\Users\z-perkins-thomas\Documents\bin\ruby\whitewidow>```

**Comment by Ekultek** [comment]
Fixed via 06db61a
